### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jkauffman503**
Updating both ReadMe and Contributing to have consistent formatting

### CODE REVIEW

**Comment by jkauffman503** [comment]
@shawnbutts @majormoses @Createor I wanted to take a hack at updating the Contributing.md and README.md files to be consistent with our our other GitHub projects. Please let me know if you have any feedback, or if you liked some things they way they used to be.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,82 @@
...
+ 
+Workflow
+------------------------------
+ 
+1. Create a branch directly in this repo or a fork (if you don't have push access). Please name
+branches within this repository `feature/<description>` or `fix/description`. For example, 
+something like `feature/upgrade_agent_0.2.3-70`.
 
-1. Create a branch directly in this repo or a fork (if you don't have push access). Please name branches within this repository `feature/<description>` or `fix/description`. For example, something like `feature/upgrade_agent_0.2.3-70`.
+2. Create an issue or open a pull request (PR). If you aren't sure your PR will solve the issue
```
Comment (majormoses) [inline CONTRIBUTING.md]: why manually keep track of numbers in github md format 1. will automatically find the correct number

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,26 +1,82 @@
...
+or may be controversial, we're okay with you opening an issue separately and linking to it in 
+your PR. That way, if the PR is not accepted, the issue will remain and be tracked.
 
-1. Create an issue or open a PR. If you aren't sure your PR will solve the issue, or may be controversial, we commend opening an issue separately and linking to it in your PR, so that if the PR is not accepted, the issue will remain and be tracked.
+3. Clone the fork/branch locally:
 
-1.  Close (and reference) issues by the `closes #XXX` or `fixes #XXX` notation in the commit message. Please use a descriptive, useful commit message that could be used to understand why a particular change was made.
+    
+        $ git clone https://github.com/Netuitive/chef-netuitive.git
```
Comment (majormoses) [inline CONTRIBUTING.md]: maybe its just me but I feel like its outside of the scope to do a git tutorial here...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,26 +1,82 @@
...
+    
+4. Close (and reference) issues by the `closes #XXX` or `fixes #XXX` notation in the commit
+message. Please use a descriptive, useful commit message that could be used to understand why a
+particular change was made.  <b>Example:</b>
 
-1. Keep pushing commits to the initial branch, using `--amend`/`--rebase` as  necessary. Please don't mix fixing unrelated issues in a single branch.
+        
+        $ git add *.txt
```
Comment (majormoses) [inline CONTRIBUTING.md]: see above comment

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,26 +1,82 @@
...
+    * Passed continuous integration (CI) tests
+    * Version bumps
+    * A changelog
+    
+8. After further review, a project maintainer can merge your request.
 
-1. After reviewing commits for documentation, passing CI tests, version bumps, changelog, and good descriptive commit messages, a maintainer can merge.
-
-
-### Releasing
```
Comment (majormoses) [inline CONTRIBUTING.md:21]: Why are we taking this out?

File: README.md [hunk 5]
```diff
@@ -1,82 +1,86 @@
...
+### Official
+Debian 8, Ubuntu 14.04 LTS, CentOS 6.5, and CentOS 7. Automated testing will be performed to ensure coverage of these platforms.
 
-### Not officially Supported
+### Unofficial
 We will attempt to support as many linux distributions as possible and are hoping to expand the above list over time. Any EPEL based system that still supports yum will likely work and we are open to PRs to expands functionality.
 
+## Using the Netuitive Cookbook
 
-## Attributes
```
Comment (majormoses) [inline README.md:18]: Why are we moving attributes down to the bottom? I think this is the most common thing someone would come to the readme for beyond initial set up.

**Comment by majormoses** [comment]
@jkauffman503 I think there are definitely some good changes here though I think we will want it slightly different to follow the common patterns of chef repos. I am looking into why CI started failing all of a sudden...

**Comment by majormoses** [comment]
@jkauffman503 I have fixed the CI issues: #21 ...as soon as this is merged please rebase my changes into your branch.

**Comment by majormoses** [comment]
@jkauffman503 I self merged it as it was only CI fixes. please rebase to make your CI pass

File: CONTRIBUTING.md (hunk 2 above)
Comment (jkauffman503) [inline CONTRIBUTING.md]: You're probably right. I'll go ahead and remove this command as well as the one below.

File: CONTRIBUTING.md (hunk 4 above)
Comment (jkauffman503) [inline CONTRIBUTING.md:21]: I reworked this into a different bullet, but the original bullet says it better. I'll change what I have.

File: README.md (hunk 5 above)
Comment (jkauffman503) [inline README.md:18]: I'll move this up to the top below the description (where it was before).

**Comment by majormoses** [comment]
@jkauffman503 this looks better, are you done making changes if so lets squash those commits down...

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,26 +1,70 @@
...
-1. When everything is ready for merge, clean up the branch (rebase with master to synchronize, squash, edit commits, etc) to prepare for it to be merged.
+1. After reviewing your commits for documentation, passed continuous integration (CI) tests, 
+version bumps, changelogs, and good, descriptive commit messages, a project maintainer can merge your request.
 
-### Merging contributions
-
-1. After reviewing commits for documentation, passing CI tests, version bumps, changelog, and good descriptive commit messages, a maintainer can merge.
-
-
-### Releasing
```
Comment (majormoses) [inline CONTRIBUTING.md:21]: Id like to bring this back in

**Comment by majormoses** [comment]
@jkauffman503 any chance you could be done by tomorrow? I would like to merge #22   but I need to add doc and I want to wait until we have made the format solid.

**Comment by jkauffman503** [comment]
It'll be done by tomorrow!

Sent using CloudMagic Emailhttps://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.4.40&pv=5.1.1&source=email_footer_2
On Thu, Apr 28, 2016 at 8:02 PM, Ben Abrams <notifications@github.com<mailto:notifications@github.com>> wrote:

@jkauffman503https://github.com/jkauffman503 any chance you could be done by tomorrow? I would like to merge #22https://github.com/Netuitive/chef-netuitive/pull/22 but I need to add doc and I want to wait until we have made the format solid.

##

**Comment by jkauffman503** [comment]
Finished rebase and merge with remote. It's a little ugly, but I think it's ready.

**Comment by majormoses** [comment]
closing this and moving it to: https://github.com/Netuitive/chef-netuitive/pull/23

**Pull request closed without merging**